benchmarks/lockhammer/scripts/lh_test_cfg.yaml

```diff
@@ -34,8 +34,8 @@
 
 ## Global Settings
 globalcfg:
-    execdir: /tmp/synchronization-benchmarks/benchmarks/lockhammer/build
-    logfile: /tmp/lockhammer.csv
+    execdir: /root/synchronization-benchmarks/benchmarks/lockhammer/build
```

Comment:
You cannot hard-code the paths for the unit-tests.  These paths need to be automatically generated in the code to be relative to wherever the code is being run from, either in Jenkins or someone's workstation.